config/crds/README.md

```diff
@@ -1 +1,16 @@
+**Managing Troubleshoot CRDs**
 
+The CRDs in this folder are used by Troubleshoot in combination with the spec that can be passed on to the Troubleshoot CLI in various ways (the location of the spec can be passed on as a web address, yaml file or a secret residing on a cluster).
```

Comment:
You mention the spec here but never again. Is the spec relevant here, if not I'd not bother mentioning it, if so some clarification on why it's important would be good. I don't think how a spec is passed to the cli is particularly relevant to the CRDs.

Reply:
I think this statement is describing why there are CRDs and what they're used for  - in this case parsing specs for various Troubleshoot components.  If that's the case, this could be reworded to make the intent of the CRDs clearer and where they're used in the code (or, at least, what for).  I'd avoid talking about the CLI specifically unless the CRDs are _only_ relevant when Troubleshoot is called from the CLI, but I don't think that's the case?